src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/SqlIaaSExtensionTests.cs

```diff
@@ -15,6 +15,7 @@ public SqlIaaSExtensionTests(Xunit.Abstractions.ITestOutputHelper output)
         public void TestSqlIaaSExtension()
         {
             ComputeTestController.NewInstance.RunPsTest("Test-SetAzureRmVMSqlServerExtension");
+            ComputeTestController.NewInstance.RunPsTest("Test-SetAzureRmVMSqlServerAKVExtension");
```

Comment:
Same comment as before - please edit the function in the ps1 script to include the new cmdlets, or please add a new test method for thsi new extension.